crates/tabby-download/Cargo.toml

```diff
@@ -11,4 +11,5 @@ reqwest = { version = "0.11.18", features = ["stream", "json"] }
 anyhow = { workspace = true }
 serde = { workspace = true }
 serdeconv = { workspace = true }
+tracing = { worksapce = true }
```

Comment:
Is there a typo here? (`worksapce` instead of `workspace`)

Reply:
nice catch! fixing in pr https://github.com/TabbyML/tabby/pull/331